test/symlink.js

```diff
@@ -2,6 +2,7 @@ var resolve = require("../");
 var should = require("should");
 var path = require("path");
 var fs = require("fs");
+var os = require("os");
```

Comment:
`package.json` states that the node version is >= 6.9.0 for this module. there's no reason new code should be using `var` instead of `let` or `const`

also, if the only thing we're using `os` for is `platform` (see below) then use destructuring here.